test/functional/apps/visualize/_tile_map.js

```diff
@@ -171,33 +171,33 @@ export default function ({ getService, getPageObjects }) {
        ** check some data after we save the viz, then zoom in and check that the data
        ** changed, then open the saved viz and check that it's back to the original data.
        */
-      it('should save with zoom level and load, take screenshot', async function () {
-
-        const expectedZoom5Data = [
-          `- 9q5 91 { "lat": 34.2934322102855, "lon": -118.57068326651722 }`,
-          `- 9qc 89 { "lat": 38.64546895785822, "lon": -121.59105236401383 }`,
-          `- 9qh 74 { "lat": 34.18319454366291, "lon": -117.426273193009 }`,
-          `- 9y7 73 { "lat": 35.87868071952197, "lon": -96.3330221912275 }`,
-          `- 9ys 71 { "lat": 37.31065319536228, "lon": -94.82038319412567 }`,
-          `- 9yn 71 { "lat": 34.57203017311617, "lon": -92.17198946946104 }`,
-          `- 9q9 70 { "lat": 37.327310177098425, "lon": -121.70855726221842 }`,
-          `- 9zv 65 { "lat": 44.39887339514322, "lon": -93.34879252386207 }`,
-          `- 9yu 65 { "lat": 38.83211174621605, "lon": -94.76601768729205 }`,
-          `- 9y6 64 { "lat": 35.743489960441366, "lon": -97.75019769280334 }`
-        ];
-
-        const expectedZoom6Data = [
-          `- c20g 16 { "lat": 45.59211894578766, "lon": -122.47455075674225 }`,
-          `- c28c 13 { "lat": 48.0181491561234, "lon": -122.43847891688347 }`,
-          `- c40b 11 { "lat": 56.34347582997923, "lon": -133.61804031343624 }`,
-          `- c2e5 11 { "lat": 48.46440218389034, "lon": -119.51805034652352 }`,
-          `- c262 10 { "lat": 46.56816971953958, "lon": -120.5440594162792 }`,
-          `- c23n 10 { "lat": 47.51524904742837, "lon": -122.26747375912964 }`,
-          `- 9rw6 10 { "lat": 42.59157135151327, "lon": -114.79671782813966 }`,
-          `- c2mq 9 { "lat": 47.547698873095214, "lon": -116.18850083090365 }`,
-          `- c27x 9 { "lat": 47.753206375055015, "lon": -118.7438936624676 }`,
-          `- c25p 9 { "lat": 46.30563497543335, "lon": -119.30418533273041 }`
-        ];
+      it('zoom in to top level, check required behaviors on the way dwn', async function () {
+
+        // const expectedZoom5Data = [
```

Comment:
Can you link to the issue again and explain about why it's all commented out?  Unless i don't see it, I think you had that in here before.

So the sort and prepData fixed the other functions but still not this one?

Reply:
yes, I'll llnk the issue again.

the prepData is just to simplify the code a little, while I was working on it. but those resolved errors i got locally, and have never seen in CI.